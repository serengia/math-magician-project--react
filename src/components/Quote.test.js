// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";

import Quote from "./Quote";

it("render quotes correctly", () => {
  const quoteObj = {
    quote: "Test quote",
    by: "Mandela",
  };
  const tree = renderer.create(<Quote quoteObj={quoteObj} />).toJSON();
  expect(tree).toMatchSnapshot();
});
