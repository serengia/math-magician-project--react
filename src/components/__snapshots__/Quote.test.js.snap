// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render quotes correctly 1`] = `
<div
  className="math-quote"
  style={
    Object {
      "backgroundColor": "#fff",
    }
  }
>
  <p
    className="quote-text"
  >
    Test quote
  </p>
  <span
    className="by"
  >
    By: 
    <span
      className="name"
    >
      Mandela
    </span>
     
  </span>
</div>
`;
